.github/workflows/ci.yml

```diff
@@ -41,23 +41,10 @@ jobs:
       - name: Code sniff
         run: composer run-script code:lint
 
-      - name: Install Terminus
-        run: |
-          wget https://github.com/pantheon-systems/terminus/releases/download/3.0.0-beta1/terminus
-          chmod +x terminus
-          mv terminus /usr/local/bin/t3
-          mkdir ~/.terminus
-          t3 self:info
-
-      - name: Install Terminus Build Tools
-        run: |
-          t3 self:plugin:install terminus-build-tools-plugin
```

Comment:
These are used so in some of the Robo workflows. Is there some other place that the build tools are being installed?

Reply:
It's in the docker image